src/Logger/LogsHttpLogger.php

```diff
@@ -0,0 +1,188 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\logs_http\Logger\LogsHttpLogger.
+ */
+
+namespace Drupal\logs_http\Logger;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Logger\LogMessageParserInterface;
+use Drupal\Core\Logger\RfcLogLevel;
+use Drupal\Core\Logger\RfcLoggerTrait;
+use Psr\Log\LoggerInterface;
+
+class LogsHttpLogger implements LoggerInterface {
```

Comment:
You should create `LogsHttpLoggerInterface`, so if someone changes the service to their own implementation of `LogsHttpLoggerInterface` this module will still work. and then you will have

```php
interface LogsHttpLoggerInterface extends LoggerInterface {}

class LogsHttpLogger extends LogsHttpLoggerInterface {}
```